### ISSUE TITLE
Scroll script not working

### ISSUE THREAD

**Issue by andreasarf** [HtmlUnit/htmlunit#332]
As I understood, `window.scrollBy` function should be working as result of https://github.com/HtmlUnit/htmlunit/issues/282. Apparently, it is not the case when I tried it in my case to trigger some lazy load using v2.48.0:

```java
try (final WebClient webClient = new WebClient(BrowserVersion.BEST_SUPPORTED)) {
      HtmlPage htmlPage = webClient.getPage("https://www.tokopedia.com/p/handphone-tablet/handphone?page=1");
      System.out.println(htmlPage.getByXPath("//div[@data-testid='lstCL2ProductList']/div").size());
      htmlPage.executeJavaScript("window.scrollBy(0,2000);");
      webClient.waitForBackgroundJavaScript(10_000);
      System.out.println(htmlPage.getByXPath("//div[@data-testid='lstCL2ProductList']/div").size());
} catch (IOException e) {
      
}
```
The number of elements after scroll is the same as before.

What did I miss? Is the scroll functionality still not perfect?

**Comment by rbri** [comment]
Yes you are right, the function is supported with the current version. And i can reproduce your problem - executing the sniped from the dev tools works but not with HtmlUnit.

But this is one page with a lot of js magic inside - maybe the scrolling relates somewhere on the current layout.
At the moment i see no chance to find the time to dig deeper into this to figure out what is going wrong.
So if you can do some more analysis i will like to help but i can't do the deep debugging for this at the moment.
